.codesandbox/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:lts-alpine
+
+# 1. zsh is default shell on CodeSandbox - without it, the devbox fails to start
+# with a "/bin/zsh: no such file or directory" error:
+#
+# ```
+# OCI runtime exec failed: exec failed: unable to start container process: exec: '/bin/zsh': stat /bin/zsh: no such file or directory: unknown
+#
+# ```
+#
+# 2. Git is a dependency of zsh - without it, opening the terminal fails with a
+# "command not found" error:
+# ```
```

Comment:
```suggestion
# ```
#
# 2. Git is a dependency of zsh - without it, opening the terminal fails with a
# "command not found" error:
#
# ```
```